kustomize/base/deployment-jira-mcp.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
```

Comment:
This deployment should be in its own namespace, idea being it can be commonly reused by others also to avoid tight coupling with BugZooka.